content/attribute_persistence.md

```diff
@@ -23,15 +23,15 @@ In Chef Infra Client 16.3 the node Blacklist and Whitelist features were renamed
 
 Legacy config mapping:
 
-- automatic_attribute_blacklist -> blocked_automatic_attributes
```

Comment:
The point here was to call out the old / new attributes so the old ones have to be in the docs.